guidelines/licenses.md

```diff
@@ -0,0 +1,9 @@
+# License
+
+You can select license during repository creation. If you forgot doing that then follow this [instruction](https://help.github.com/en/articles/adding-a-license-to-a-repository).
+
+If `year` is required use current year value.
+
+If `organization` is required use following value: `Sandstream Development Sp. z o. o.`
+
+You can also use one of [GitHub templates](https://github.com/licenses/license-templates/blob/master/templates/), fill required fields and add to repository under `LICENSE` filename in root of the project.
```

Comment:
Consider providing a default template for the MIT license, with organization data.